tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html

```diff
@@ -200,6 +200,16 @@
   stroke: var(--tb-graph-faded);
 }
 
+#unfilled-rect {
+  stroke: #a6a6a6;
+}
+
+#devices-checkbox
+{
```

Comment:
nit: Could this block be styled like the one above? ie, curly brace after selector and space after each colon.